definitions/npm/rxjs_v5.0.x/flow_>=v0.29.0/rxjs_v5.0.x.js

```diff
@@ -260,6 +207,238 @@ declare module 'rxjs' {
+      c: Observable<C>,
+    ): Observable<[A, B, C]>;
+
+    static combineLatest<A, B, C, D>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+    ): Observable<[A, B, C, D]>;
+
+    static combineLatest<A, B, C, D, E>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+    ): Observable<[A, B, C, D, E]>;
+
+    static combineLatest<A, B, C, D, E, F>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+      f: Observable<F>,
+    ): Observable<[A, B, C, D, E, F]>;
+
+    static combineLatest<A, B, C, D, E, F, G>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+      f: Observable<F>,
+      g: Observable<G>,
+    ): Observable<[A, B, C, D, E, F, G]>;
+
+    static combineLatest<A, B, C, D, E, F, G, H>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+      f: Observable<F>,
+      g: Observable<G>,
+      h: Observable<H>,
+    ): Observable<[A, B, C, D, E, F, G, H]>;
+
+    combineLatest<A>(
+      a: Observable<A>
+    ): Observable<[T, A]>;
+
+    combineLatest<A, B>(
+      a: Observable<A>,
+      resultSelector: (a: A) => B,
+    ): Observable<B>;
+
+    combineLatest<A, B, C>(
+      a: Observable<A>,
+      b: Observable<B>,
+      resultSelector: (a: A, b: B) => C,
+    ): Observable<C>;
+
+    combineLatest<A, B, C, D>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      resultSelector: (a: A, b: B, c: C) => D,
+    ): Observable<D>;
+
+    combineLatest<A, B, C, D, E>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      resultSelector: (a: A, b: B, c: C, d: D) => E,
+    ): Observable<E>;
+
+    combineLatest<A, B, C, D, E, F>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+      resultSelector: (a: A, b: B, c: C, d: D, e: E) => F,
+    ): Observable<F>;
+
+    combineLatest<A, B, C, D, E, F, G>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+      f: Observable<F>,
+      resultSelector: (a: A, b: B, c: C, d: D, e: E, f: F) => G,
+    ): Observable<G>;
+
+    combineLatest<A, B, C, D, E, F, G, H>(
+      a: Observable<A>,
+      b: Observable<B>,
+      c: Observable<C>,
+      d: Observable<D>,
+      e: Observable<E>,
+      f: Observable<F>,
+      g: Observable<G>,
+      resultSelector: (a: A, b: B, c: C, d: D, e: E, f: F, g: G) => H,
+    ): Observable<H>;
+
+    withLatestFrom<A>(
+      a: Observable<A>
+    ): Observable<[T, A]>;
+
+    withLatestFrom<A, B>(
+      a: Observable<A>,
+      resultSelector: (a: A) => B,
```

Comment:
Is there some reason that `t: T` was removed from the resultSelector? I needed to add it back in order to get my flow checks to pass.